example-adapter/Setting-up-Development-Environment.md

```diff
@@ -2,24 +2,24 @@ This document describes how a developer's workstation can be setup.
 
 ### Software Requirements
 
-* Linux or Windows
-* Java 7
-* Tomcat 6 or 7 or 8
+* Linux / Windows / MacOS
+* Java 8 or Java 11
 * Maven 3.x
+* Docker (*optional*)
 
 ### Getting the code
 
-The example adapter project is a part of the XRd4J git repository. You can access the source code by cloning it:
+The Example Adapter project is a part of the XRd4J git repository. You can access the source code by cloning it:
 
 ```
 git clone https://github.com/nordic-institute/xrd4j.git
 ```
 
-The example adapter is a separate Maven project and can be found under the `example-adapter` directory.
+The Example Adapter is a separate Maven project and can be found under the `example-adapter` directory.
 
 ### Building the code
 
-Example adapter uses Maven as the build management tool. In order to build the whole project and generate the WAR-file (`example-adapter-x.x.x-SNAPSHOT.war`), you must run the maven command below from the project root directory.
+Example Adapter uses Maven as the build management tool. In order to build the whole project and generate the WAR-file (`example-adapter-x.x.x-SNAPSHOT.war`), you must run the maven command below from the project root directory.
 
 ```
 mvn clean install
```

Comment:
I think this could be `mvn clean verify package` instead. I believe that `install` also installs the package to the local maven repository, which probably is not required in this case. `package` should just create the jar/war files and leave it at that. `verify` is needed because it appears that just `package` does not run checks, e.g. the dependency checker.

Reply:
Changed this to `mvn clean verify` since installing the package to local maven repository isn't required in this case. However, `verify` already includes `package` so there's no need to include `package` explicitly.